src/MergePlugin.php

```diff
@@ -151,7 +151,7 @@ public function uninstall(Composer $composer, IOInterface $io)
      */
     public static function getSubscribedEvents()
     {
-        if (defined('InstallerEvents::PRE_OPERATIONS_EXEC')) {
+        if (defined('\Composer\Installer\InstallerEvents::PRE_OPERATIONS_EXEC')) {
```

Comment:
Technically, the leading \ inside such string FQCNs shouldn't be needed, though.